src/resources/MachineLearning/CaseClassificationConfiguration/tests/CaseClassificationConfiguration.spec.ts

```diff
@@ -1,14 +1,13 @@
 import API from '../../../../APICore';
-import {New} from '../../../BaseInterfaces';
-import CCConfiguration from '../CaseClassificationConfiguration';
+import CaseClassificationConfiguration from '../CaseClassificationConfiguration';
 import {CaseClassificationConfigurationModel, Operator} from '../CaseClassificationConfigurationInterfaces';
 
 jest.mock('../../../../APICore');
 
 const APIMock: jest.Mock<API> = API as any;
 
 describe('CCConfiguration', () => {
```

Comment:
For consistency, I would suggest to update the describe title to `CaseClassificationConfiguration` as well.

Reply:
yes good idea